Gamma_Analysis.py

```diff
@@ -43,13 +40,16 @@ def absolute_efficiency(energy, coeffs=EFFICIENCY_CAL_COEFFS):
     return efficiency
 
 
-def emission_rate(net_area, efficiency, livetime):
+def emission_rate(net_area, livetime):
```

Comment:
So what you're saying is `emission_rate` has an identity crisis and other functions can't relate to it the same way anymore. Hmmm...

I'm intrigued, would definitely like to work on this simpler method of finding the isotope activity for the sample and reference. Alright, code reverted!